docker-entrypoint.sh

```diff
@@ -116,6 +116,10 @@ if [[ "$RECORDLOG_ENABLED" ]]; then
     JQ_FILTERS_CONFIG="$JQ_FILTERS_CONFIG | .recordLog.enabled=true"
 fi
 
+if [[ "$STORAGE_QUOTA_ENABLED" ]]; then
```

Comment:
Can you change the name to `STORAGE_LIMIT_ENABLED`? It is the official name of the feature and will avoid confusion with the upcoming quotas support (per user).